galaxy_api/api/v3/viewsets.py

```diff
@@ -30,6 +30,9 @@
 from galaxy_api.common import pulp
 from galaxy_api.api import permissions, models
 
+import logging
```

Comment:
@alikins I've already created a PR adding `logging`, the order should be updated: https://github.com/ansible/galaxy-api/pull/114#discussion_r342482673

Feel free to close my PR if this is merged first.

Reply:
@romanblanco I merged #114, and rebased this pr